conda/common/io.py

```diff
@@ -32,6 +32,10 @@
 _FORMATTER = Formatter("%(levelname)s %(name)s:%(funcName)s(%(lineno)d): %(message)s")
 
 
+def dashlist(iterable, indent=2):
+    return ''.join('\n' + ' ' * indent + '- ' + str(x) for x in iterable)
```

Comment:
The point in putting this here now is to start transitioning to using it from here.  We've always been importing it from `conda.resolve` before, which is so wrong.